app/templates/praktici_admin.html

```diff
@@ -0,0 +1,146 @@
+                        </td>
+                        <td>
+                            <input type="text" name="poznamka[]" value="{{ vaccine.poznamka }}"/>
+                        </td>
+                    </tr>
+                    {% endfor %}
+                    </tbody>
+                </table>
+            </div>
+            <button type="submit" class="btn btn-primary">Aktualizovat</button>
+            <a class="btn btn-warning" href="{{ url_for('view.praktici_admin_logout') }}">Odhlásit se</a>
+        </form>
+    </div>
+    {% else %}
+    <div class="col-sm-6 mb-3">
+        <div class="card card-body bg-light">
+            <h5>Registrovat se</h5>
+            <form action="{{ url_for('view.praktici_admin_register') }}" method="post">
+                <div class="form-group row">
+                    <label for="register-id" class="col-lg-4 col-form-label">Kód ordinace</label>
+                    <div class="col-lg-8">
+                        <input type="text" class="form-control" id="register-id" name="id">
+                    </div>
+                </div>
+                <div class="form-group row">
+                    <div class="col-lg-4"></div>
+                    <div class="col-lg-8">
+                        <button type="submit" class="btn btn-primary">Registrovat</button>
+                    </div>
+                </div>
+            </form>
+        </div>
+    </div>
+    <div class="col-sm-6 mb-3">
+        <div class="card card-body bg-light">
+            <h5>Editovat své záznamy</h5>
+            <form action="{{ url_for('view.praktici_admin_login') }}" method="post">
+                <div class="form-group row">
+                    <label for="login-id" class="col-lg-4 col-form-label">Kód ordinace</label>
+                    <div class="col-lg-8">
+                        <input type="text" class="form-control" id="login-id" name="id">
+                    </div>
+                </div>
+                <div class="form-group row">
+                    <label for="login-passwd" class="col-lg-4 col-form-label">Heslo</label>
+                    <div class="col-lg-8">
+                        <input type="password" class="form-control" id="login-passwd" name="passwd">
+                    </div>
+                </div>
+                <div class="form-group row">
+                    <div class="col-lg-4"></div>
+                    <div class="col-lg-8">
+                        <button type="submit" class="btn btn-primary">Přihlásit</button>
+                    </div>
+                </div>
+            </form>
+            <p class="card-text font-italic">
+                Pokud jste heslo zapomněli, napište nám o něj na TODO.
+            </p>
+        </div>
+    </div>
+    {% endif %}
+</div>
+
+<div class="row">
+    <div class="col">
+        <h4>Přehled</h4>
+        {{ volne_vakciny(all_vaccines) }}
```

Comment:
Remove ?

Reply:
i would leave it there to see change immediately